src/packages/package.json

```diff
@@ -29,6 +29,11 @@
       "tar-fs@2.1.1": "2.1.3",
       "tar-fs@3.0.8": "3.0.9"
     },
-    "onlyBuiltDependencies": ["websocket-sftp", "websocketfs", "zeromq"]
+    "onlyBuiltDependencies": [
+      "http-proxy-3",
```

Comment:
I'm confused -- why did you add http-proxy-3 to onlyBuiltDependencies?  It doesn't even involve compiled code -- it's all javascript.